src/Makefile.in

```diff
@@ -984,10 +984,20 @@ peflags:
 	@echo Securing executables.
 	$(PEFLAGS) $(PROJ)
 
+###############################################################################
+# To use it, run
+# $ make -s windows-tiny-link-to-john MAIN_NAME=john-avx2.exe
+# We intentionally do not use CFLAGS here: must not have this binary depend on
+# any non-guaranteed SIMD instruction set.
```

Comment:
We can use `CFLAGS_MAIN` which contains everything except things like `-mAVX2` and is used for `john.o` (but it does define eg. `JOHN_AVX2` just so we can know what the other files get).